packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_def.hpp

```diff
@@ -1969,12 +1969,15 @@ namespace MueLu {
      auto sym = rcp(new Tpetra::CrsGraphTransposer<LocalOrdinal,GlobalOrdinal,Node>(tpGraph));
      auto tpGraphSym = sym->symmetrize();
 
-     auto rowsSym = tpGraphSym->getNodeRowPtrs();
+     auto colIndsSym =        // temporary; better fix would be change to LWGraph constructor
```

Comment:
maybe the comment should say FIXME, so if we ever get bored we can grep for it along with Mark's other similar comments?

Reply:
Done.